tests/linearizability/model.go

```diff
@@ -197,6 +224,24 @@ func initState(request EtcdRequest, response EtcdResponse) EtcdState {
 		case Put:
 			state.KeyValues[op.Key] = op.Value
 		case Delete:
+		case PutWithLease:
+			if _, ok := state.Leases[op.LeaseID]; ok {
```

Comment:
This should never happen as `state.Leases` should be empty.

Reply:
updated in https://github.com/etcd-io/etcd/pull/15093